Game/Levels/FilterWorld/L04_mem_principal_self.lean

```diff
@@ -25,7 +25,7 @@ TheoremDoc MyGame.mem_principal_self as "mem_principal_self" in "Filter"
 -- TODO needs proof! Also needs hints for new users
 /-- A set `A` is an element of the principal filter `𝓟 A`. -/
 Statement mem_principal_self (A : Set 𝓧) : A ∈ 𝓟 A := by
-  sorry
+rw[mem_principal]
```

Comment:
Level 4 needs the TODOs removed, a better introduction (for example
```
Introduction "Here's another fact about principal filters, which isn't
one of the axioms of a filter, but is handy to have around: it's
part of a robust \"API\" for principal filters (i.e., standard lemmas
which are useful to have). See if you can prove it yourself."
```

and an explanation in the `Conclusion` about why `rw` closed the goal, for example
```
Conclusion "Wooah! If you did `rw [mem_principal]` then the goal was closed
automatically! This is because `rw` tries `rfl` afterwards, and `⊆` is reflexive
so the tactic works.

Let's now talk about an order on filters."
```